infrastructure/local/docker-compose.yml

```
@@ -1,7 +1,9 @@
 version: '3'
 services:
   auth:
-    image: ghcr.io/interledger/rafiki-auth:latest
+    build:
+      context: ../..
+      dockerfile: ./packages/auth/Dockerfile
```

This seems to also work
```suggestion
      context: ../../packages/auth
```

---

nvm
That worked when pulling, but context does need to be in the repo root when building